package.json

```diff
@@ -39,8 +39,8 @@
   },
   "babel": {
     "presets": [
-      "es2015",
-      "react"
+      "@babel/env",
```

Comment:
This [part](https://babeljs.io/docs/en/env/#basic-options-docs-en-options-change) in the migration note is incorrect. I got the following error when I `npm start`

```js
Module build failed (from ./node_modules/babel-loader/lib/index.js):
Error: Cannot find module 'babel-preset-env'
- Did you mean "@babel/env"?
```

So I changed the following two lines to `@babel/env` and `@babel/react`